lib/config/folderconfiguration.go

```diff
@@ -90,27 +92,51 @@ func (f *FolderConfiguration) CreateMarker() error {
 		return nil
 	}
 
-	permBits := fs.FileMode(0o777)
-	if build.IsWindows {
-		// Windows has no umask so we must chose a safer set of bits to
-		// begin with.
-		permBits = 0o700
-	}
```

Comment:
If this is no longer needed, it should be noted in the commit message.  I suspect it stems from the time we transitioned from a marker file to a marker directory?

Isn't the same logic needed for the content file then?

Reply:
It's based on a misunderstanding to begin with. The only thing these bits do on Windows is set the read only flag, which we don't do here. Having the defaults remove group & other write bits on unixes is best practice. I'll make a note of it.